pkgs/tools/misc/nginx-config-formatter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "2016-06-16";
+  name = "nginx-config-formatter-${version}";
```

Comment:
Sorry, I didn't notice this earlier. If this is not an official release then the name should be `nginx-config-formatter-unstable-${version}` as per https://nixos.org/nixpkgs/manual/#sec-package-naming.